pkg/components/state/pluggable_test.go

```diff
@@ -650,7 +650,9 @@ func TestComponentCalls(t *testing.T) {
 		}
 		request := &state.QueryRequest{
 			Query: query.Query{
-				Filters: filters,
+				QueryFields: query.QueryFields{
+					Filters: filters,
+				},
```

Comment:
What's this doing here?

Reply:
@berndverst explained why this is here, nevermind.

Reply:
The Query interface was changed in contrib.